repl/evaluator.go

```diff
@@ -430,7 +431,12 @@ type Evaluator struct {
 
 // NewEvaluator returns an inialized evaluator
 func NewEvaluator() (*Evaluator, error) {
-	env, err := cel.NewEnv()
+	env, err := cel.NewEnv(
+		ext.Strings(),
+		ext.Protos(),
+		ext.Maths(),
```

Comment:
add a simple test that the base environment includes these extensions

Reply:
Done